### PR TITLE
first working implementation obj supporting

### LINKED ISSUE #655: Begin native .obj parser into Object3D
FYI @pavletto  @maxbebop

### PR DESCRIPTION
**Pull request by pavletto**

### CODE REVIEW

File: src/og/Object3d.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 import { htmlColorToFloat32Array } from './utils/shared.js';
 import { Vec3 } from './math/Vec3.js';
-import { MIN, MAX } from './math.js';
+import { MAX, MIN } from './math.js';
+import { convertToWebGLSpace, objParser } from "./utils/objParser.js";
```
Comment (Zemledelec) [inline src/og/Object3d.js]: `transformLeftToRightCoordinateSystem` better

File: src/og/utils/objParser.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+    // same order as `f` indices
+    const objVertexData = [
+        objPositions,
+        objTexcoords,
+        objNormals,
+    ];
+
+    // same order as `f` indices
+    let webglVertexData = [
```
Comment (Zemledelec) [inline src/og/utils/objParser.js]: no need _webgl_ prefix

**Review by Zemledelec — CHANGES_REQUESTED** [review]
Change leftToRightCoordinateSystem

File: src/og/Object3d.js (hunk 1 above)
Comment (pavletto) [reply src/og/Object3d.js]: Fixed

File: src/og/utils/objParser.js (hunk 2 above)
Comment (pavletto) [reply src/og/utils/objParser.js]: Fixed

**Review by Zemledelec — APPROVED** [review]

**Comment by pavletto** [comment]
@Zemledelec ready to finish review. Sadenly I commented ray intersection test - check it please.

**Review by Zemledelec — APPROVED** [review]

**Pull request merged**